float32 distancia
float32 angulo

string nombre